### DISCUSSION TITLE
Classification : using different thresholds for 2 models in ensemble

### DISCUSSION THREAD

**Discussion by DipanjanSanyal** [pycaret/pycaret#1255] [Q&A] [answered]
I have a binary classification problem. Model 1 is logistic but I need to change threshold (unbalanced data, want to try without SMOTE, threshold chart suggesting much lower threshold than 0.5). Model 2 is xgboost with default threshold. Final model is ensemble_model(Model1, Model2). I am not able to set custom threshold for Model 1 only, because custom threshold is part of predict_model, not part of saved model pipeline. Is this really not possible or am I missing out something?

**Comment by Yard1** [comment, marked as answer]
This is not yet possible.
